packages/workspace/src/builders/run-commands/schema.json

```diff
@@ -48,7 +48,12 @@
           }
         }
       ]
+    },
+    "cwd": {
+      "type": "string",
+      "description": "Current working directory of the commands."
     }
   },
-  "required": ["commands"]
+  "required": ["commands"],
+  "additionalProperties": false
```

Comment:
@jaysoo - was there any reason we were not restricting `additionalProperties` before? Let me know if you can think of any issues with me adding this

Reply:
I think we will want this to be true so we can in theory pass extra params and forward them to the underlying command. We don't do it now, but we should

Reply:
I see - yes, I initially thought we can use the `args` option for that - but I can see how it's a different use-case..`args` is more powerful / allows interpolation throughout the command and needs to be set-up beforehand in workspace.json

while allowing extra options would be simpler if one would want to just forward them to the underlying command like you say e.g. `nx generate-docs frontend --option1="foo" --option2="bar"`

Reply:
We can do that as a follow-up feature/PR where we do arguments forwarding.